spec/unit/provider/package/pip3_spec.rb

```diff
@@ -2,282 +2,15 @@
 require 'spec_helper'
 
 provider_class = Puppet::Type.type(:package).provider(:pip3)
-osfamilies = { ['All', nil] => ['pip3'] }
 
 describe provider_class do
 
-  before do
-    @resource = Puppet::Resource.new(:package, "fake_package")
-    @provider = provider_class.new(@resource)
-    @client = stub_everything('client')
-    @client.stubs(:call).with('package_releases', 'real_package').returns(["1.3", "1.2.5", "1.2.4"])
-    @client.stubs(:call).with('package_releases', 'fake_package').returns([])
-    XMLRPC::Client.stubs(:new2).returns(@client)
+  it "should inherit most things from pip provider" do
+    expect(provider_class < Puppet::Type.type(:package).provider(:pip))
   end
 
-  describe 'provider features' do
```

Comment:
What's not inherited? The pip3 implementation is really small, levering almost everything from the pip provider. Why have duplicated tests?